frontend/src/renderer/pages/config/Config.tsx

```diff
@@ -37,10 +38,9 @@ function saveConfig(OPENAI_API_KEY: string, model: string, embeddingsmodel: stri
 //             console.log('Error fetching config:', error);
 //             return false;
 //         });
-// }
+// }console.log("saveconfig");
     return apiCall('/config', 'POST', payload).then((response) => {
```

Comment:
Would you say we should extend the `apiCall` function with a boolean to deactivate the Notification? As of now a user gets the Endpoint error + the configerror.